phoenixRest/models/crew/card_order.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    DateTime,
+)
+from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.postgresql import UUID, ENUM
+
+from datetime import datetime
+import uuid
+import enum
+
+class OrderStates(str, enum.Enum):
+    CREATED     = "CREATED"
+    IN_PROGRESS = "IN_PROGRESS"
+    FINISHED    = "FINISHED"
+    CANCELLED   = "CANCELLED"
+    
+from phoenixRest.models import Base
```

Comment:
this is just a nitt pick but maybe move the import over the class like in my suggestion
```suggestion
from phoenixRest.models import Base

class OrderStates(str, enum.Enum):
    CREATED     = "CREATED"
    IN_PROGRESS = "IN_PROGRESS"
    FINISHED    = "FINISHED"
    CANCELLED   = "CANCELLED"
```